docs/fundamentals/connection.txt

```diff
@@ -0,0 +1,25 @@
+.. _laravel-fundamentals-connection:
+
+===========
+Connections
+===========
+
+.. toctree::
+
+   /fundamentals/connection/connect-to-mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+Learn how to set up a connection and specify connection behavior from your
+Laravel application to a MongoDB deployment by using {+odm-short+} in the
+following sections:
+
+- :ref:`Connect to MongoDB <laravel-connect-to-mongodb>`
```

Comment:
q: why is this called connect to MongoDB instead of Connection Guide?

Reply:
This was a custom title with similar intent to the suggestions you made earlier. To keep it simple, I'd suggest that we only use the titles of the pages and then describe them as necessary. Let me know what you think and I'll add the appropriate task to the cleanup ticket.

Reply:
This sounds like a good solution to prevent the same page from being referred to by different things in different places!

Reply:
Great, added "Use the original titles for all landing page link lists. Use a description if it needs additional explanation."